### PR TITLE
feat: calculate time travel

### PR DESCRIPTION
**Pull request by gabitrujillo**
## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Review by MorveN11 — COMMENTED** [review]
This pull request is empty, please check your changes before creating a pull request

File: apps/client/src/app/pages/map/hooks/useCalculateTravelTime.tsx
Comment (Severaldog) [inline apps/client/src/app/pages/map/hooks/useCalculateTravelTime.tsx:1]: The file is empty

File: apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  end: LngLat
+): Promise<number> => {
+  const response = await fetch(
+    `https://api.mapbox.com/directions/v5/mapbox/cycling/${start.lat},${
+      start.lng
+    };${end.lat},${end.lng}?access_token=${
+      import.meta.env.VITE_PUBLIC_MAPBOX_TOKEN
+    }`
+  );
+  interface DirectionsResponse {
```
Comment (MorveN11) [inline apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx:15]: Please move this interface to the globals types

File: apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { LngLat } from 'mapbox-gl';
+
+const calculateTravelTime = async (
+  start: LngLat,
+
+  end: LngLat
+): Promise<number> => {
+  const response = await fetch(
```
Comment (MorveN11) [inline apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx:8]: please consider move this fetch to a service

File: apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { LngLat } from 'mapbox-gl';
+
+const calculateTravelTime = async (
+  start: LngLat,
+
```
Comment (MorveN11) [inline apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx:5]: remove this unnecessary enter

File: apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { LngLat } from 'mapbox-gl';
+
+const calculateTravelTime = async (
+  start: LngLat,
```
Comment (MorveN11) [inline apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx:4]: start of what, be a more explicit what the names of your variables

File: apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import { LngLat } from 'mapbox-gl';
+
+const calculateTravelTime = async (
```
Comment (MorveN11) [inline apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx:3]: consider to put your variables in keys

File: apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  start: LngLat,
+
+  end: LngLat
+): Promise<number> => {
+  const response = await fetch(
+    `https://api.mapbox.com/directions/v5/mapbox/cycling/${start.lat},${
```
Comment (MorveN11) [inline apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx:9]: Does the application only calculate the travel time in cycling? please try to create a enum for this

**Review by MorveN11 — COMMENTED** [review]
Please review all these comments before merging your branch :D

**Pull request closed without merging**